src/Symfony/Component/Translation/Loader/IcuResFileLoader.php

```diff
@@ -39,7 +39,6 @@ public function load($resource, $locale, $domain = 'messages')
         try {
             $rb = new \ResourceBundle($locale, $resource);
         } catch (\Exception $e) {
-            // HHVM compatibility: constructor throws on invalid resource
```

Comment:
You mean this one? https://github.com/symfony/symfony/pull/23216#discussion_r122739646 I answered.